app/src/main/java/org/fossasia/susi/ai/dataclasses/GroupWiseSkills.kt

```diff
@@ -0,0 +1,12 @@
+package org.fossasia.susi.ai.dataclasses
+
+import org.fossasia.susi.ai.rest.responses.susi.SkillData
+
+/**
+ *
+ * Created by arundhati24 on 16/07/2018.
+ */
+data class GroupWiseSkills(
+        var group: String,
+        var skillsList: MutableList<SkillData>
```

Comment:
Why are these var, not val

Reply:
Because these are being assigned values when we get response from the server.